### PR TITLE
Better accomodate helm on left and right. 

### PR DESCRIPTION
**Pull request by ralesi**
Autoresize doesn't work well on the side.

### CODE REVIEW

File: spacemacs/packages.el [hunk 1]
```diff
@@ -1491,7 +1491,9 @@ Example: (evil-map visual \"<\" \"<gv\")"
     :config
     (progn
       (when dotspacemacs-helm-resize
-        (setq helm-autoresize-min-height 1)
+                  (or (eq dotspacemacs-helm-position 'bottom)
```
Comment (syl20bnr) [inline spacemacs/packages.el]: Are you missing an `and` here ?

**Comment by ralesi** [comment]
Shoot,  yes it didnt make it over from the other commit.  I will fix it.

**Comment by ralesi** [comment]
Sorry about that.  It's fixed now.

**Comment by syl20bnr** [comment]
Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**